theme.yaml

```diff
@@ -21,3 +21,6 @@ original:
   name: al-folio
   homepage: https://alshedivat.github.io/al-folio/
   repo: https://github.com/alshedivat/al-folio
+
+params:
+  recent_posts_count: 3
```

Comment:
Do you have a reference for this? I can't find any. And where is it used?

Reply:
No where. I had an idea which is not fully implemented. Let's delete it for now.

Reply:
I deleted this one and added a new variable in `params.yaml`. It shows recent blogs.